src/app/operators/dialogs/nature40-catalog/nature40-catalog.component.html

```diff
@@ -0,0 +1,34 @@
+<wave-sidenav-header>Nature 4.0 Catalog</wave-sidenav-header>
+<ng-container [ngSwitch]="(catalog$ | async)?.size">
+    <div *ngSwitchCase="undefined" class="error">
+        <mat-spinner></mat-spinner>
+    </div>
+    <div *ngSwitchCase="0" class="error mat-typography"><p>The catalog is currently empty.</p></div>
```

Comment:
is the error class the correct one for loading / empty catalogs?

Reply:
164cfcf